mk/linux.mk

```diff
@@ -23,9 +24,15 @@ COVERAGE=--coverage
 endif
 endif
 
+ifeq ($(HARDWARE_ARCH),aarch64)
+ARCHCFLAGS=-march=armv8-a+crc+crypto
```

Comment:
It turns out that all aarch64 CPUs have SIMD instructions, so there's no need to tell the compiler it can use them - it just will.  The [Amazon Graviton](https://aws.amazon.com/ec2/graviton/) processors used in EC2 aarch64 instances also support the CRC and Crypto instructions, so I told the compiler it can use them.  These were added to the ARMv8 architecture fairly early on, so should not restrict usage too much.  I suspect any hardware that didn't support these instructions would be unable to run ML well anyway, e.g. a 5 year old Raspberry Pi.